qiskit_ionq/ionq_backend.py

```diff
@@ -125,7 +125,7 @@ class IonQBackend(Backend):
 
     @classmethod
     def _default_options(cls):
-        return Options(shots=1024)
+        return Options(shots=1024, api_settings=None)
```

Comment:
What do you think if we rename to job_settings ? I think that'd be closer to the intent.